### ISSUE TITLE
Localize the dashes in time ranges

### ISSUE THREAD

**Issue by mitchellevan** [ThePacielloGroup/babel#38]
### Currently

Demo pages are using en dashes for time ranges, without spaces. This is [correct for US and UK English](https://proofed.co.uk/writing-tips/en-dashes-and-em-dashes-in-us-and-uk-english/).

However, at least Spanish should not use en dashes for time ranges.

This is important because screen reader pronounciation varies by context for dashes and hyphens.

### Proposed

Find out what the range symbols should be, and change them where needed.

* [ ] Spanish
* [x] German
* [ ] Lower Sorbian
* [ ] Javanese
* [ ] template instructions

#### Spanish

Use a hyphen.

guion, hyphen https://estilotraduccion.blogspot.com/2013/11/guion-raya-barra-y-asterisco.html?m=1

With spaces around it, " - " ? Examples:

https://www.aldi.es/supermercados/encuentra-tu-supermercado.html/l/madrid/calle-de-la-costa-brava-36/3604392

https://www.dia.es/tiendas/buscador-tiendas/madrid/pozuelo-de-alarcon/28023/282

https://www.tiendeo.com/Tiendas/madrid/zara-calle-gran-via/104195

Wrong character, but has spaces:

https://biblioagenda.ucm.es/hours

Or without spaces around it? Examples:

https://www.usc.gal/es/servicios/area/biblioteca-universitaria/horarios-bibliotecas

#### German

https://de.wikipedia.org/wiki/Halbgeviertstrich

https://www.sekretaria.de/bueroorganisation/korrespondenz/geschaeftsbriefe/uhrzeit-richtig-schreiben/

Example:
Montag 09:30 – 16:30 Uhr

#### Lower Sorbian

?

#### Javanese

?

**Comment by mitchellevan** [comment]
I used hyphens for Spanish for the reasons cited, and en dashes for the others.
